pandas/core/config_init.py

```diff
@@ -455,12 +456,27 @@ def is_terminal() -> bool:
     ``future.infer_string`` is set to True.
 """
 
+
+def is_valid_string_storage(value: Any) -> None:
```

Comment:
Do we need a test with a bad storage argument?

Reply:
That already exists (I could add a test specifically for "pyarrow_numpy")

Reply:
Ah sorry I missed it. Fine to skip another test for `pyarrow_numpy`